pom.xml

```diff
@@ -442,7 +339,8 @@
         </dependency>
         <dependency>
             <groupId>org.codehaus.groovy</groupId>
-            <artifactId>groovy-test</artifactId>
+            <artifactId>groovy-all</artifactId>
```

Comment:
Looks like this caused a cascade of breaking references in about 12 projects. Here's the output from the logs:

```
ERROR] The build could not read 12 projects -> [Help 1]
[ERROR]   
[ERROR]   The project  (/home/travis/build/apache/nifi/nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-nar/pom.xml) has 1 error
[ERROR]     Non-parseable POM /home/travis/build/apache/nifi/nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-nar/pom.xml: end tag name </project> must match start tag name <dependencies> from line 32 (position: TEXT seen ...</dependency>\n</project>... @51:11)  @ line 51, column 11 -> [Help 2]
[ERROR]   
[ERROR]   The project org.apache.nifi:nifi-json-utils:1.10.0-SNAPSHOT (/home/travis/build/apache/nifi/nifi-commons/nifi-json-utils/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-json:jar is missing. @ line 38, column 21
[ERROR]   
[ERROR]   The project org.apache.nifi:nifi-web-security:1.10.0-SNAPSHOT (/home/travis/build/apache/nifi/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-json:jar is missing. @ org.apache.nifi:nifi-web-security:[unknown-version], /home/travis/build/apache/nifi/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/pom.xml, line 180, column 21
[ERROR]   
[ERROR]   The project org.apache.nifi:nifi-standard-processors:1.10.0-SNAPSHOT (/home/travis/build/apache/nifi/nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml) has 2 errors
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-json:jar is missing. @ org.apache.nifi:nifi-standard-processors:[unknown-version], /home/travis/build/apache/nifi/nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml, line 374, column 21
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-servlet:jar is missing. @ org.apache.nifi:nifi-standard-processors:[unknown-version], /home/travis/build/apache/nifi/nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml, line 379, column 21
[ERROR]   
[ERROR]   The project org.apache.nifi:nifi-lookup-services:1.10.0-SNAPSHOT (/home/travis/build/apache/nifi/nifi-nar-bundles/nifi-standard-services/nifi-lookup-services-bundle/nifi-lookup-services/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-json:jar is missing. @ org.apache.nifi:nifi-lookup-services:[unknown-version], /home/travis/build/apache/nifi/nifi-nar-bundles/nifi-standard-services/nifi-lookup-services-bundle/nifi-lookup-services/pom.xml, line 166, column 21
[ERROR]   
[ERROR]   The project org.apache.nifi:nifi-mongodb-processors:1.10.0-SNAPSHOT (/home/travis/build/apache/nifi/nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-json:jar is missing. @ org.apache.nifi:nifi-mongodb-processors:[unknown-version], /home/travis/build/apache/nifi/nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/pom.xml, line 113, column 21
[ERROR]   
[ERROR]   The project org.apache.nifi:nifi-scripting-bundle:1.10.0-SNAPSHOT (/home/travis/build/apache/nifi/nifi-nar-bundles/nifi-scripting-bundle/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-all:pom is missing. @ org.apache.nifi:nifi:1.10.0-SNAPSHOT, /home/travis/build/apache/nifi/pom.xml, line 340, column 21
[ERROR]   
[ERROR]   The project org.apache.nifi:nifi-elasticsearch-client-service:1.10.0-SNAPSHOT (/home/travis/build/apache/nifi/nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-json:jar is missing. @ org.apache.nifi:nifi-elasticsearch-client-service:[unknown-version], /home/travis/build/apache/nifi/nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/pom.xml, line 163, column 21
[ERROR]   
[ERROR]   The project org.apache.nifi:nifi-groovyx-bundle:1.10.0-SNAPSHOT (/home/travis/build/apache/nifi/nifi-nar-bundles/nifi-groovyx-bundle/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-all:pom is missing. @ org.apache.nifi:nifi:1.10.0-SNAPSHOT, /home/travis/build/apache/nifi/pom.xml, line 340, column 21
[ERROR]   
[ERROR]   The project org.apache.nifi:nifi-cdc-mysql-processors:1.10.0-SNAPSHOT (/home/travis/build/apache/nifi/nifi-nar-bundles/nifi-cdc/nifi-cdc-mysql-bundle/nifi-cdc-mysql-processors/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-json:jar is missing. @ org.apache.nifi:nifi-cdc-mysql-processors:[unknown-version], /home/travis/build/apache/nifi/nifi-nar-bundles/nifi-cdc/nifi-cdc-mysql-bundle/nifi-cdc-mysql-processors/pom.xml, line 68, column 21
[ERROR]   
[ERROR]   The project org.apache.nifi:nifi-mock-record-utils:1.10.0-SNAPSHOT (/home/travis/build/apache/nifi/nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-mock-record-utils/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-json:jar is missing. @ org.apache.nifi:nifi-mock-record-utils:[unknown-version], /home/travis/build/apache/nifi/nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-mock-record-utils/pom.xml, line 55, column 21
[ERROR]   
[ERROR]   The project org.apache.nifi:nifi-toolkit-encrypt-config:1.10.0-SNAPSHOT (/home/travis/build/apache/nifi/nifi-toolkit/nifi-toolkit-encrypt-config/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.codehaus.groovy:groovy-cli-commons:jar is missing. @ org.apache.nifi:nifi-toolkit-encrypt-config:[unknown-version], /home/travis/build/apache/nifi/nifi-toolkit/nifi-toolkit-encrypt-config/pom.xml, line 114, column 21
```

Reply:
That commit is mid-progress (noted in the commit message).  I'll be pushing another change today to resolve all that and shift around some of the groovy-* dependencies.